cmake/CxxQt.cmake

```diff
@@ -41,9 +41,14 @@ function(cxx_qt_generate_cpp GEN_SOURCES)
 
     file(MAKE_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/target/cxx-qt-gen")
 
+    get_target_property(QMAKE Qt${QT_VERSION_MAJOR}::qmake IMPORTED_LOCATION)
+
     # Run cargo during config to ensure the cpp source file list is created
     execute_process(
-        COMMAND ${CARGO_CMD}
+        COMMAND cmake -E env
```

Comment:
add some comments here as to why we need and what are CARGO_TARGET_DIR and QMAKE